astropy/io/fits/tests/test_core.py

```diff
@@ -34,6 +34,10 @@ def test_with_statement(self):
     def test_missing_file(self):
         fits.open(self.temp('does-not-exist.fits'))
 
+    def test_missing_file(self):
```

Comment:
The test should have a somewhat more descriptive name! (and not overwrite the previous one...)

Reply:
Oups ... thanks for spotting that!